tutorials/hyperspectral/RasterStack-RGB-Images-in-R-Using-HSI.Rmd

```diff
@@ -301,15 +276,15 @@ We can play with the color ramps too if we want:
 
 #change the colors of our raster 
```

Comment:
Add space between the # and the text in these comments to standardize throughout.

Reply:
Ooof yes this was my mistake and it was widespread. I went through and corrected all herein.